### DISCUSSION TITLE
java agent do not collect full logs of app？

### DISCUSSION THREAD

**Discussion by hasaiki123** [apache/skywalking#12092] [Q&A] [answered]
version：
skywalking apm 9.3.0
java agent 9.1.0

I want to collect app logs to skywalking oap server，and after I configure all，like logback conf。I found the logs collected，and it can be showed in the web ui，but it looks like not full logs，and when I try to produce more logs，it do not collect more，I found there is not full logs in database。And，if I restart my app，it can collect log again，but also if I try to produce more logs，it do not collect again。

Is there any problem？Please help me，thanks。

**Comment by wu-sheng** [comment, marked as answer]
The description is not enough to address the issues. You could check logs and maybe whether data is sent through network.

**Comment by hasaiki123** [comment]
I found some error configuration in logbcak-spring.xml.
Thanks a lot.
